src/scss/components/header/_my-lists.scss

```diff
@@ -0,0 +1,147 @@
+@import "./../../base/variables";
```

Comment:
if you will add in app.scss your import you don't need import again variables and mixins inside your component. They are visible automatically.